### PR TITLE
add tests for trajectory transforms

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
lgtm

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
           python -Ic "import openfe_analysis; print(openfe_analysis.__version__)"
       - name: "Run tests"
         run: |
-          pytest -n auto -v --cov=openfe_analysis --cov-report=xml --durations=10
+          pytest -n 1 -v --cov=openfe_analysis --cov-report=xml --durations=10
```
Comment (mikemhenry) [inline .github/workflows/ci.yaml:56]: Having some issues with xdist?

**Review by mikemhenry — APPROVED** [review]

**Comment by mikemhenry** [comment]
Turned on branch protection and required pull requests.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (richardjgowers) [reply .github/workflows/ci.yaml:56]: Yeah the nc files don't play well, had a crashed worker

**Pull request merged**